3.Tests/UI/StepDefinitions/Settings/Profile/UpdateFullNameStepDefinition.cs

```diff
@@ -8,18 +8,21 @@ namespace SeleniumTest.Tests;
 [Scope(Feature = "Full Name update")]
```

Comment:
I already remove this file in a new PR so you can omit these changes please.

Reply:
Done